OpenSim/Common/C3DFileAdapter.h

```diff
@@ -68,29 +68,37 @@ class OSIMCOMMON_API C3DFileAdapter : public FileAdapter {
         C3DFileAdapter c3dFileAdapter;
         c3dFileAdapter.setLocationForForceExpression(C3DFileAdapter::ForceLocation::CenterOfPressure);
         auto tables  =  c3dFileAdapter.read("myData.c3d");
+        auto markersTable = c3dFileAdapter.getMarkersTable(tables);
+        auto forcesTable = c3dFileAdapter.getForcesTable(tables);
         \endcode
 
         <b>Python example</b>
         \code{.py}
         import opensim
-        C3DFileAdapter c3dFileAdapter;
+        c3dFileAdapter = opensim.C3DFileAdapter()
         c3dFileAdapter.setLocationForForceExpression(opensim.C3DFileAdapter.ForceLocation_CenterOfPressure);
         tables = c3dFileAdapter.read("myData.c3d")
+        markersTable = c3dFileAdapter.getMarkersTable(tables)
+        forcesTable = c3dFileAdapter.getForcesTable(tables)
         \endcode
 
         <b>Java example</b>
         \code{.java}
         C3DFileAdapter c3dFileAdapter = new C3DFileAdapter();
         c3dFileAdapter.setLocationForForceExpression(C3DFileAdapter.ForceLocation.CenterOfPressure);
         tables = c3dFileAdapter.read("myData.c3d");
-                    
+        TimeSeriesTableVec3 markersTable = c3dFileAdapter.getMarkersTable(tables);
+        TimeSeriesTableVec3 forcesTable = c3dFileAdapter.getForcesTable(tables);
+
         \endcode
 
         <b>MATLAB example</b>
         \code{.m}
          C3DFileAdapter c3dFileAdapter;
          c3dFileAdapter.setLocationForForceExpression(1);
-        tables = C3DFileAdapter.read("myData.c3d");
+         tables = C3DFileAdapter.read("myData.c3d");
```

Comment:
When did Matlab switch to double quotes? What version are you using?

Reply:
Thanks for the catch @tkuchida, I didn't change the line but I believe you're correct. I made the change accordingly.

Reply:
Ah, ok thanks. I don't have Matlab hooked up but maybe a Matlab user could test this.